common/config/pprof.go

```diff
@@ -65,10 +66,15 @@ func (initializer *PProfInitializerImpl) Start() error {
 		return nil
 	}
 
+	host := initializer.PProf.Host
+	if host == "" {
+		host = "localhost"
+	}
+
 	if atomic.CompareAndSwapInt32(&pprofStatus, pprofNotInitialized, pprofInitialized) {
 		go func() {
-			initializer.Logger.Info("PProf listen on ", tag.Port(port))
-			err := http.ListenAndServe(fmt.Sprintf("localhost:%d", port), nil)
+			initializer.Logger.Info("PProf listen on ", tag.Host(host), tag.Port(port))
```

Comment:
host tag might be confused with the host name of the service which is appended by logging agents usually. so let's just use `tag.Dynamic("pprof-host", host)` here

Reply:
Ok, changed to `tag.Dynamic("pprof-host", host)` and remove `Host` function from tag .